packages/react/src/components/DotcomShell/DotcomShell.js

```diff
@@ -25,7 +25,10 @@ const DotcomShell = ({ navigation, children }) => {
     <>
       <div className={`${prefix}--grid ${prefix}--dotcom-shell`}>
         <div className={`${prefix}--dotcom-shell__masthead`}>
-          <Masthead navigation={navigation} />
+          <>
```

Comment:
I'm sorry, but what is this doing exactly?

Reply:
Passes nav data to the masthead component